libs/novui/src/components/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,23 @@
+import { MouseEventHandler, ReactNode } from 'react';
+import { MdClose } from 'react-icons/md';
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Better to get the icon from our icons registry in novui‏